src/interfaces/ISettings.sol

```diff
@@ -12,6 +12,10 @@ interface ISettings {
     /// @param enabled: true if account execution is enabled, false if disabled
     event AccountExecutionEnabledSet(bool enabled);
 
+    /// @notice emitted when a token is added to or removed from the whitelist
+    /// @param token: address of the token
+    event TokenWhitelistStatusUpdated(address token);
```

Comment:
**Suggestion:** Add `_isWhitelisted` to this event so it is easy to see what happened.

Reply:
Done !